Assets/Scripts/GameloopControler.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class GameloopControler : MonoBehaviour
+{
+    [SerializeField] private PlayerManager playerManager;
+    [SerializeField] private ObstacleManager obstacleManager;
+
+    private Action onPlayerDied;
+
+    void Start()
+    {
+        AssetsDatabase.Init();
+        obstacleManager.Init("Obstacle");
+        playerManager.Init("Player");
+        onPlayerDied += StopGame;
+        playerManager.SubscribePlayer(onPlayerDied);
+    }
```

Comment:
This works fine, but how about create a method just to subscribe to events, and another to unsubscribe from them?